src/CoreVideo/CVPixelBuffer.cs

```diff
@@ -147,7 +181,7 @@ public static CVPixelBuffer Create (nint width, nint height, CVPixelFormatType p
 
 		class PlaneData
 		{
-			public GCHandle[] dataHandles;
+			public GCHandle[]? dataHandles;
```

Comment:
The `dataHandles` field is set to a non-null value as soon as a `PlaneData` instance is created, so logically it should always be non-null:

https://github.com/xamarin/xamarin-macios/blob/ae6577ed3c2e353b42ba67347cd6eb4b709cb73e/src/CoreVideo/CVPixelBuffer.cs#L263-L264

Reworking this so that the `dataHandles` field is set in a constructor would make it possible to make the field non-nullable (and avoid the other related changes below).